getting-started/openshift/installation.md

```diff
@@ -85,6 +85,18 @@ curl {{ "/manifests/ocp/tigera-operator/02-tigera-operator.yaml" | absolute_url
 curl {{ "/manifests/ocp/01-cr-installation.yaml" | absolute_url }} -o manifests/01-cr-installation.yaml
 ```
 
+#### Add install time resources
+
+Add any [Calico resources]({{site.baseurl}}/reference/resources) that need to be created at startup to
+`manifests/02-configmap-calicoctl-resources.yaml`.
```

Comment:
Also, I know this is already sorta merged... but I think we shouldn't call these "calicoctl-resources". They are Calico API objects or Calico resources. `calicoctl` is just a tool you can use to provision them. Can we rename?

Reply:
I like Calico API objects, I avoided Calico resources because I thought they could be confused with the kubernetes calico resources (like the node daemonset, configmaps, and all that jazz).

Reply:
After chatting with Barbara she suggested that Calico resources is the wording we've used other places so that is what I've gone with. In the 02-configmap-calico-resources.yaml I added a line to clarify that only projectcalico.org/v3 resources should be added to that file. In the documentation the link to the Calico resources reference should be enough clarification there.